### PR TITLE
Update test data genny to create phase 1 sjq and bq or just bq tests

### PR DESCRIPTION
**Pull request by benjstephenson**
based on guaranteed interview status.

### CODE REVIEW

File: app/services/testdata/Phase1TestsInvitedStatusGenerator.scala [hunk 1]
```diff
@@ -18,37 +18,54 @@ package services.testdata
...
+  override val gatewayConfig = cubiksGatewayConfig
 }
 
 trait Phase1TestsInvitedStatusGenerator extends ConstructiveGenerator {
   val otRepository: OnlineTestRepository
+  val gatewayConfig: CubiksGatewayConfig
 
   def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier) = {
 
+    val sjq = Phase1Test(
```
Comment (mgilgar) [inline app/services/testdata/Phase1TestsInvitedStatusGenerator.scala]: What is sjq?

File: app/services/testdata/Phase1TestsInvitedStatusGenerator.scala [hunk 2]
```diff
@@ -18,37 +18,54 @@ package services.testdata
...
+      token = UUID.randomUUID().toString,
+      testUrl = generatorConfig.cubiksUrl,
+      invitationDate = generatorConfig.phase1StartTime.getOrElse(DateTime.now()).withDurationAdded(86400000, -1),
+      // TODO: Add started datetime
+      participantScheduleId = 149245,
+      scheduleId = gatewayConfig.onlineTestConfig.scheduleIds("sjq"),
+      usedForResults = true
+    )
+
+    val bq = Phase1Test(
```
Comment (mgilgar) [inline app/services/testdata/Phase1TestsInvitedStatusGenerator.scala]: What is bq?

**Comment by benjstephenson** [comment]
These are the two Cubiks tests that candidates get invited to in phase1 testing - they're referenced like that consistently across the applications.

**Comment by mgilgar** [comment]
@benjstephenson Can you add Test to bq, sqj? like bqTest or sqjTest or bqExercise or sqjExercise?

**Pull request merged**